### DISCUSSION TITLE
Helper to click on recycler view item by name

### DISCUSSION THREAD

**Discussion by martin-stephens** [AdevintaSpain/Barista#380] [Ideas]
For example, the barista assertion could be: `fun clickOnItem(recyclerViewId: Int, targetText: String)`

The implementation:
`onView(withId(recyclerViewId)).perform(RecyclerViewActions.actionOnItem(hasDescendant(withText(targetText)), click()));`

**Comment by Sloy** [comment]
Hi @smxt! Thanks for the suggestion! It sounds like something Barista could have indeed.

We already have `clickListItem(R.id.list, 4)` to click a RecyclerView or ListView by position. And overloaded function with a String parameter would make sense, similar to the `clickOn()` functions.

Would you like to contribute this feature? I suspect the implementation would be very similar to those in `clickListItem`
